docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,567 @@
+---
+title: "Build a bedtime songs bot with a custom ML model"
+linkTitle: "Bedtime Songs Bot"
+weight: 60
+type: "docs"
+tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
+description: "Create a robot babysitter with a webcam and machine learning."
+image: "/tutorials/bedtime-songs-bot/export-screenshot.png"
```

Comment:
```suggestion
```
I don't think it needs "image" but idk whether that's related to the build error or not

Reply:
It doesn't need image or imageAlt. Those are an either or with the webmsrc, mp4src and videoAlt. Essentially either a preview card on a page has a video or an image